src/features/runtime-checks.js

```diff
@@ -66,12 +66,20 @@ const jsMimeTypes = [
     'text/x-javascript'
 ]
 
+function getTaintFromScope (scope, args, shouldStackCheck = false) {
+    try {
+        scope = args.callee.caller
+    } catch {}
+    return hasTaintedMethod(scope, shouldStackCheck)
+}
+
 class DDGRuntimeChecks extends HTMLElement {
```

Comment:
not directly related to this PR, but with more functionality being added we should be discussing when it makes sense to split this file up.